### ISSUE TITLE
Make projects "trackable". 

### ISSUE THREAD

**Issue by NekoJonez** [GlotPress/GlotPress-WP#1252]
So, I think it would be awesome if you could "track" projects. So, what do I mean here....

Well, I use several plugins on my website that update quite often and add new strings to translate. Instead of having a whole pile of bookmarks and checking them from time to time, I think it would be handy if you could "track" projects.

The idea would be that when you select your locale and project, you would click a button that adds the project on your track list. 

I would add a tab after "Stats" that said: "Tracked" or something of that nature. On this page, projects that aren't 100% completed anymore due to an update would show up, and you could finish their translation. 

The idea is a bit rough and it should be first designed and such... But I think it could improve and help polygots to keep projects 100% translated after an update instead of always moving on to the next project. (Which CAN be a double edged sword, I'm well aware of that.)

**Comment by ocean90** [comment]
Since I know you're coming from translate.w.org: Did you know that you can mark themes/plugins as favorites on their w.org pages? The list is then used for the default filter in the themes/plugins list. Example:

![image](https://user-images.githubusercontent.com/617637/124600766-5b518900-de67-11eb-88d8-143d6cdc396a.png)

Both favorites have untranslated strings so they are on top of that list.

**Comment by NekoJonez** [comment]
Yes, I do know that :) But, that will only show the things per tab... It would be lovely to have them combined into one tab. Plus, the ability to choose meta projects for it.

I know certain other polygots are using this method already but it would be better if this feature is expanded upon imho.
